### ISSUE TITLE
page request: fossil

### ISSUE THREAD

**Issue by pixelcmtd** [tldr-pages/tldr#6458]
https://fossil-scm.org/

**Comment by sbrl** [comment]
Thanks for requesting this page! Do you want to have a go yourself?

Pages are best created by people who understand the command they are about.

If you do, take a quick read of [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md), which guides you through the process of creating a new page here. Then, just simply send a pull request to this repository - and if you need any assistance, just ask :-)

**Comment by pixelcmtd** [comment]
@sbrl Could you please stop copy-pasting the same message below every page request? If we want this spammed, I'm sure, we can set up a bot or just change the damn Issue template.

**Comment by CleanMachine1** [comment]
I think its more for presentation of issues for anyone browsing thats new.

**Comment by marchersimon** [comment]
@pixelcmtd I have to agree with @ CleanMachine1  here. Also, in regard of spamming, I don't think we need a page request for every page we're adding. Just opening the PR would be enough.

**Comment by pixelcmtd** [comment]
> I think its more for presentation of issues for anyone browsing thats new.

Then it should be part of the Issue template.
